src/main/java/core/basesyntax/Commands.java

```diff
@@ -0,0 +1,75 @@
+package core.basesyntax;
+
+import java.io.File;
+import java.util.Scanner;
+
+public class Commands implements Utilities {
+
+    private String command;
+
+    private Scanner scanner;
+
+    public void appStart() {
+        greeting();
+        scanner = new Scanner(System.in);
+        appListening();
+    }
+
+    public void appListening() {
+        command = scanner.nextLine();
+        try {
+            commandManager();
+        } catch (IndexOutOfBoundsException e) {
+            System.out.println("Команда требует иное количество аргументов");
+            appListening();
+        } catch (RuntimeException e) {
+            System.out.println("Веденная информация неверна!");
+        }
+    }
+
+    public void commandManager() {
+        if (!command.contains("/")) {
+            appendTextToFile(command);
+            appListening();
+        } else {
+            String[] commandData = command.split(" ");
+            switch (commandData[0]) {
```

Comment:
@YaTut1901 read about Command pattern and Strategy pattern here: https://refactoring.guru/